src/Pages/Explore/helper.js

```diff
@@ -0,0 +1,16 @@
+export const searchVideos=(videos,searchVideo)=>{
+   let updatedList=[...videos];
```

Comment:
Here I think you dont need to spread  `videos` because filter will anyway give you a new array.